basic-features.md

```diff
@@ -1,80 +1,136 @@
+| [List COLUMNS partitioning](/partitioned-table.md)                                                       | Experimental | Experimental | N            | N            | N            | N            |
+| **Statistics**                                                                                           | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [CMSketch](/statistics.md)                                                                               | Deprecated   | Deprecated   | Y            | Y            | Y            | Y            |
+| [Histograms](/statistics.md)                                                                             | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Extended statistics (multiple columns)](/statistics.md)                                                 | Experimental | Experimental | N            | N            | N            | N            |
+| **Security**                                                                                             | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Transparent layer security (TLS)](/enable-tls-between-clients-and-servers.md)                           | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Encryption at rest (TDE)](/encryption-at-rest.md)                                                       | Y            | Y            | Y            | N            | N            | N            |
+| [Role-based authentication (RBAC)](/role-based-access-control.md)                                        | Y            | Y            | Y            | Y            | Y            | N            |
+| [Certificate-based authentication](/certificate-authentication.md)                                       | Y            | Y            | Y            | Y            | Y            | N            |
+| Support for MySQL 8.0 clients                                                                            | Y            | Y            | Y            | N            | N            | N            |
+| [MySQL compatible `GRANT` system](/privilege-management.md)                                              | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Dynamic Privileges](/privilege-management.md#dynamic-privileges)                                        | Y            | N            | N            | N            | N            | N            |
+| [Security Enhanced Mode](/system-variables.md#tidb_enable_enhanced_security)                             | Y            | N            | N            | N            | N            | N            |
+| [Redacted Log Files](/log-redaction.md)                                                                  | Y            | Y            | N            | N            | N            | N            |
```

Comment:
@dveeden Can you please review this section (Security)?

Reply:
I don't really know much about pre-4.0 releases, but this all seems correct.

We _could_ link to `security-compatibility-with-mysql.md` and add a note about support for MySQL password hash formats `mysql_native_password` and `caching_sha2_password`.

Reply:
I intentionally did not document `caching_sha2_password` yet. If we document master, it will be hard to keep track of experimental vs. stable, as it changes so much leading up to release.